graylog2-web-interface/src/components/datanode/hooks/useShowDatanodeMigration.ts

```diff
@@ -27,7 +29,9 @@ const useShowDatanodeMigration = () : boolean => {
     fetchShowDatanodeMigration,
   );
 
-  return isDatanodeConfiguredAndUsed === false;
+  const { currentStep } = useMigrationState();
+
+  return (isDatanodeConfiguredAndUsed === false) || (currentStep?.state !== 'FINISHED');
```

Comment:
```suggestion
  return !(isDatanodeConfiguredAndUsed && (!currentStep || currentStep?.state === 'FINISHED'));
```
we also need to consider the case that Graylog has been started with data node without ever doing a migration (i.e. regular preflight)